### PR TITLE
T5

### PR DESCRIPTION
**Pull request by owsym**

### CODE REVIEW

File: task/t5.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+from utils.reader import file_content_t5
+
+values = file_content_t5()
+for file_value in values:
+    date = file_value[1]
+    event = file_value[22]
+
+    if event == "Rain" or event == "Snow" or event == "Rain-Snow":
```
Comment (Hamma111) [inline task/t5.py:8]: better way":
```
if events in ["Rain", "Snow", "Rain-Snow"]:

File: utils/reader.py [hunk 2]
```diff
@@ -9,3 +9,14 @@ def file_content():
 
     return file_values
 
+#function for t5
+def file_content_t5():
```
Comment (Hamma111) [inline utils/reader.py:13]: use one function to read both files

File: task/t5.py [hunk 3]
```diff
@@ -0,0 +1,9 @@
+from utils.reader import file_content_t5
+
+values = file_content_t5()
+for file_value in values:
+    date = file_value[1]
+    event = file_value[22]
```
Comment (Hamma111) [inline task/t5.py:6]: [-2] might be easier to read

**Review by Hamma111 — CHANGES_REQUESTED** [review]

File: task/t5.py (hunk 1 above)
Comment (owsym) [reply task/t5.py:8]: o k.

File: task/t5.py (hunk 3 above)
Comment (owsym) [reply task/t5.py:6]: o k.

File: utils/reader.py (hunk 2 above)
Comment (owsym) [reply utils/reader.py:13]: ok ill try

File: utils/reader.py [hunk 4]
```diff
@@ -1,11 +1,18 @@
-def file_content():
-    with open("/home/owsym/Project/Python/barq_weather_task/files/f1.csv", 'r') as file_reader:
-        content = file_reader.read()
 
-        file_values = []
-        for value in content.split("\n")[1:-1]:
-            file_values.append(value.split(","))
-        # print(file_values)
+def read_files(file_path1, file_path2):
```
Comment (Hamma111) [inline utils/reader.py:2]: have your function which takes on paramater of file_path and return the files contents from that function. should be used like this
```python
file1_content = read_file(".../f1.csv")
file2_content = read_file(".../f2.csv")

File: task/vim-demo.txt
Comment (Hamma111) [inline task/vim-demo.txt:1]: didnt you delete these files in the last PR you merged? why are they still here?

**Review by Hamma111 — CHANGES_REQUESTED** [review]

File: task/t5.py [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+file1_path = "/home/owsym/Project/Python/barq_weather_task/files/f1.csv"
+file2_path = "/home/owsym/Project/Python/barq_weather_task/files/f2.csv"
+
+f1_File_content, f2_File_content = read_files(file1_path, file2_path)
+
+for file_value in f2_File_content:
+    date = file_value[1]
+    events = file_value[-2]
+
+    if events in ["Rain", "Snow", "Rain-Snow"]:
```
Comment (Hamma111) [inline task/t5.py:11]: good. but study `if x in y`
https://realpython.com/python-conditional-statements/#:~:text=%3E%3E%3E%20if%20%27aul%27%20in%20%27grault%27%3A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%23%20Truthy

File: utils/reader.py (hunk 4 above)
Comment (owsym) [reply utils/reader.py:2]: ok

File: task/vim-demo.txt
Comment (owsym) [reply task/vim-demo.txt:1]: i did it but idont know to came about

File: task/t5.py (hunk 5 above)
Comment (owsym) [reply task/t5.py:11]: ok

File: utils/reader.py (hunk 4 above)
Comment (owsym) [reply utils/reader.py:2]: could you please explain a little bit about how you want it?

Comment (Hamma111) [reply utils/reader.py:2]: one function to read files based on the file path

File: task/t1.py [hunk 6]
```diff
@@ -1,7 +1,7 @@
 from utils.reader import file_content
+files_directory =file_content("/home/owsym/Project/Python/barq_weather_task/files/f1.csv")
```
Comment (Hamma111) [inline task/t1.py:2]: still not relevant name. directory is folder. we dont have a dir or folder, we have the content of values Usama!!

**Review by Hamma111 — CHANGES_REQUESTED** [review]

File: task/t1.py (hunk 6 above)
Comment (owsym) [reply task/t1.py:2]: ok "file_values_content"? is correct?

Comment (Hamma111) [reply task/t1.py:2]: file_values or file_contents would be best

File: utils/reader.py [hunk 7]
```diff
@@ -1,11 +1,9 @@
-def file_content():
-    with open("/home/owsym/Project/Python/barq_weather_task/files/f1.csv", 'r') as file_reader:
-        content = file_reader.read()
+def file_content(files_directory):
```
Comment (Hamma111) [inline utils/reader.py:1]: rename this function, `get_file_content`

File: task/t1.py (hunk 6 above)
Comment (Hamma111) [inline task/t1.py:2]: file_contents = ..

**Review by Hamma111 — CHANGES_REQUESTED** [review]

File: task/t1.py (hunk 6 above)
Comment (owsym) [reply task/t1.py:2]: ok

**Pull request merged**